src/main/java/de/retest/recheck/report/ActionReplayResult.java

```diff
@@ -186,7 +185,10 @@ public String getStateFilePath() {
 		return stateFilePath;
 	}
 
-	public List<ElementDifference> getElementDifferences() {
+	/**
+	 * Return <em>all</em> child differences recursively (also child-child differences).
+	 */
+	public List<ElementDifference> getElementAllDifferences() {
```

Comment:
Maybe something like `getAllElementDifferenes` or `getDeepElementDifferences`would be more suitable.